rdf-star-cg-spec.html

```diff
@@ -558,22 +562,119 @@ <h2>Evaluation Semantics</h2>
     <section>
       <h2>Query Result Formats</h2>
 
-      <p>TODO: brief introduction paragraph (including a note that result of a CONSTRUCT query or a DESCRIBE query can be serialized using <a href="#turtle-star">Turtle*</a>)</p>
-
+      <p>In SPARQL, queries can take four forms: <em>SELECT</em>, <em>CONSTRUCT</em>, <em>DESCRIBE</em>, and <em>ASK</em> - see <a data-cite="SPARQL11-QUERY#queryForms">SPARQL1.1 Query, Section 16</a> [[SPARQL11-QUERY]]. The first of these returns a query solution as a set of variable bindings. The second and third both return an RDF graph, and the last returns a boolean value.
+      </p>
+      <p>The result of the <em>ASK</em> query form is not changed by the introduction of RDF*, and the result of the <em>CONSTRUCT</em> and <em>DESCRIBE</em> forms can be represented by <a href="#turtle-star">Turtle*</a>. However, since the <em>SELECT</em> form deals with returning individual RDF terms, the specific serialization formats for representing such query results need to be extended so that the new <a>embedded</a> triple RDF term can be represented. In this section, we propose extensions for the two most common formats for this purpose: [[[sparql11-results-json]]], and [[[rdf-sparql-XMLres]]].</p>
       <section>
         <h2>SPARQL* Query Results JSON Format</h2>
-
-        <div class="issue" data-number="13"></div>
-
+        <p>
+        The result of a SPARQL SELECT query is serialized in JSON as defined in [[[sparql11-results-json]]], which specifies a JSON representation of variable bindings to RDF terms (see [<a href="sparql11-results-json#select-results">sparql11-results-json, Section 3.2</a>]). To accomodate the new RDF term for <a>embedded</a> triples that RDF* introduces, the table of RDF term JSON representations in  <a href="sparql11-results-json#select-encode-terms">sparql11-results-json, Section 3.2.2</a> is extended with the following entry:
+        </p>
+        <dl>
+            <dt>An <a>embedded</a> triple with subject RDF term <em>S</em>, predicate RDF term <em>P</em> and object RDF term <em>O</em></dt>
+            <dd>
+              <pre>
+                {
+                  "type": "triple",
+                  "value": {
+                     "subject": <i><b>S</b></i>,
+                     "predicate": <i><b>P</b></i>,
+                     "object": <i><b>O</b></i>
```

Comment:
```suggestion
                     "subject": S,
                     "predicate": P,
                     "object": O
```
markup seems to be ignored anyway